### PR TITLE
fix(LoginView): Password field design inconsistencies

### LINKED ISSUE #6519: The password field doesn't match the design in the case of the wrong password
# Bug Report

## Steps to reproduce
1.  Open the app and enter the wrong password
2. Apply it

#### Expected behavior
The border of the Password field is becoming red
image

#### Actual behavior
The border of the Password field is not becoming red
image

### Additional Information

- Status desktop version: https://ci.status.im/job/status-desktop/job/branches/job/macos/job/master/2347/
- Operating System: macOS Monterey

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #6519

### What does the PR do
Fixed password field design inconsistencies

### Affected areas
LoginView

### Screenshot of functionality (including design for comparison)
https://user-images.githubusercontent.com/31625338/181029375-1b42132c-6f09-48ae-a412-587b25e186a9.mov

### CODE REVIEW

**Review by caybro — APPROVED** [review]
Nice!

File: ui/imports/shared/controls/Input.qml [hunk 1]
```diff
@@ -179,12 +179,13 @@ Item {
...
 
     StatusBaseText {
-        visible: !!validationError
         id: validationErrorText
+        visible: !!validationError
         text: validationError
         anchors.top: inputField.bottom
         anchors.topMargin: validationErrorTopMargin
-        anchors.right: inputField.right
+        width: parent.width
```
Comment (PascalPrecht) [inline ui/imports/shared/controls/Input.qml:187]: This looks like it's harmless, but given that you ended up changing the `width` of `Input` in the login view explicitly, I'm just asking to be sure: could this break UI in any of the other places where `Input` is used?

**Review by PascalPrecht — COMMENTED** [review]
Changes LGTM!
I'm just a bit hyper cautious with changes to a component that we use in so many places. So we should double check this won't affect other usages negatively

File: ui/imports/shared/controls/Input.qml (hunk 1 above)
Comment (alexandraB99) [reply ui/imports/shared/controls/Input.qml:187]: @PascalPrecht in text you either set a width and use horizontalAlignment or you work with contentWidth and use anchors.left/right alignment, it's the same thing.
FYI here are some screenshots from the other places the Input is used:

Comment (PascalPrecht) [reply ui/imports/shared/controls/Input.qml:187]: Cool, thanks for confirming!

**Review by PascalPrecht — APPROVED** [review]
Bonus points if we get rid off the typo in the commit message :D

**Review by saledjenic — APPROVED** [review]

**Pull request merged**